apprise/plugins/NotifyMatrix.py

```diff
@@ -1074,7 +1074,11 @@ def parse_url(url):
             'image', NotifyMatrix.template_args['image']['default']))
 
         # Get our mode
-        results['mode'] = results['qsd'].get('mode')
+        if 'mode' in results['qsd']:
+            results['mode'] = results['qsd'].get('mode')
+
+        elif 'webhook' in results['qsd']:
```

Comment:
Would you mind just rolling back this part of your change.  `mode` is consistent with all of the other plugins.  I'd rather fix this part by just updating the Wiki. The rest is good though.

The unit test is failing here anyhow because you added this block of code and no tests associated with it. Once you remove this `webhook` block, even the CI will be happy.

Reply:
Alright, then I'll revert it.